### ISSUE TITLE
Repository no longer has a Release file

### ISSUE THREAD

**Issue by lastxmas** [rkaczorek/astroberry-server#224]
I'm using an RP4 and am now getting an error on top of the certificate not trusted error that says astroberry[dot]io/repo buster Release no longer has a Release file. (paraphrasing the actual error)

**Comment by jmvm-nz** [comment]
Yeah, same issue for me, it appears the certificate issued to the website is using the wrong name, as you can see if you try manually browse to the site https://www.astroberry.io/repo/
That cert was issued for DNS Name: hosting.cdn.ovh.net and DNS Name: www.hosting.cdn.ovh.net not for astroberry.io

**Comment by lastxmas** [comment]
I was able to bypass the wrong-name issue but the no release file is something I haven't figured out how to get around yet. I'm trying to get the new indi files to cover the SVBony SV605CC camera (only SV305 is showing up in my CCD list and it doesn't recognize the camera).

**Comment by rkaczorek** [comment]
Certificate issue should be fixed shortly.
I'm not sure why would you have problems with accessing [Release file](https://www.astroberry.io/repo/dists/buster/Release)

**Comment by rkaczorek** [comment]
Try the new version https://astroberry.io/astroberry-is-back/
